source/administration/config-settings.rst

```diff
@@ -1555,9 +1584,9 @@ Custom Integrations
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 Enable Incoming Webhooks
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-Developers building integrations can create webhook URLs for channels and private groups. Please see our `documentation page <http://docs.mattermost.com/developer/webhooks-incoming.html>`_ to learn about creating webhooks, view samples, and to let the community know about integrations you have built.
+Developers building integrations can create webhook URLs for public channels and private channels. Please see our `documentation page <http://docs.mattermost.com/developer/webhooks-incoming.html>`_ to learn about creating webhooks, view samples, and to let the community know about integrations you have built.
```

Comment:
I think it's important to leave as-is, at least until people get accustomed to the terminology change from 'private groups' to 'private channels'.